iModelCore/iModelPlatform/iModelPlatform.PartFile.xml

```diff
@@ -143,7 +143,7 @@
          ********************************************************************************************** -->
 
     <!-- Entry point part -->
-    <Part Name="RunIModelEvolutionTests">
+    <Part Name="RunIModelEvolutionTests" PrgOutputDir="IModelEvolutionTests">
         <SubPart PartName="IModelEvolutionRunAllTestRunners"/>

```

Comment:
I wouldn't have thought this necessary unless posting BDFs?

Reply:
Not sure, in past `bb taglist` command was working fine in the build. Now its failing with the error described above. That's why I added it.

Reply:
Ok. I'll have to ponder that, but it's not a bad idea to have the taglist in the artifacts.